content/docs/command-reference/stage/index.md

```diff
@@ -0,0 +1,25 @@
+# stage
+
+A set of commands to add and list <abbr>stages</abbr>:
+[add](/doc/command-reference/stage/add).
```

Comment:
A bit more like https://dvc.org/doc/command-reference/cache since there's just 1 subcommand for now:
```suggestion
Contains a helper command to set the create or update <abbr>stages</abbr>:
[add](/doc/command-reference/stage/add).
```

Reply:
Wait. You mention `stage list` in the Description so I guess just put that in this intro instead of my suggestion? Or use my suggestion for now and remove the mention of `stage list` later (I assume there will be another PR for that anyway).

Reply:
Let's mention `stage list`, I'll try creating a PR today.